src/pages/register/MRegisterForm.tsx

```diff
@@ -4,7 +4,6 @@ import VMRegisterForm from './VMRegisterForm';
 
```

Comment:
const PET_INFO_REQUIRED_KEY = [
      'age',
      'name',
      'adoptionStatus',
      'type',
      tempPetInfo.weight,
      tempPetInfo.description,
      tempPetInfo.sex,
      tempPetInfo.size,
      tempPetInfo.vaccinationStatus,
      tempPetInfo.neutralizationStatus,
]

const handleChange = () => {
    setPetInfo(prev => {
        const tempPetInfo = { ...petInfo, [fieldName]: newValue };
        return {...tempPetInfo, isComplete: PET_INFO_REQUIRED_KEY.every(key => petInfo[key])}
    }, )
}
```suggestion
보고 확인해주세요...!
```